content/fundamentals/module-reference.md

```diff
@@ -20,9 +20,9 @@ export class CatsService {
 
 > info **Hint** The `ModuleRef` class is imported from the `@nestjs/core` package.
 
-#### Retrieving instances
+#### Retrieving provider instances
```

Comment:
With `get()`, you can retrieve providers, controllers, and injectables (guards, interceptors etc)